README.md

```diff
@@ -37,8 +37,10 @@ You then need to load the plugin. In `boostrap.php`, something like:
 For a complete setup, add the following to your `App\Controller\AppController`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
-public $helpers = ['BootstrapUI.Form'];
+public $helpers = [
+    'Form' => ['className' => BootstrapUI.Form'],
+    'Flash' => ['className' => 'BootstrapUI.Flash']
+];
```

Comment:
Since everything is on the view layer now (getting closer to a theme type of plugin, yay!), would you prefer suggesting the helpers to be loaded in `AppView` instead?